MTM-Frontend/src/pages/Auth/ForgotPassword.css

```diff
@@ -1,115 +1,124 @@
 /* ForgotPassword.css */
 
 @font-face {
-  font-family: Raleway;
-  src: url("../assets/Inter-VariableFont_slnt,wght.ttf");
+    font-family: Raleway;
+    src: url("../assets/Inter-VariableFont_slnt,wght.ttf");
 }
 
 .forgot-logo-image {
-  position: absolute;
-  width: 230px;
-  margin-top: 60px;
+    position: absolute;
```

Comment:
Could you fix the indentation for this file?

Reply:
Done!